pkg/reconciler/kubernetes/tektonchain/tektonchain.go

```diff
@@ -504,9 +501,9 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tc *v1alpha1.TektonChain
 	if err != nil {
 		secretInstallerSetSigningKey = false
 	}
-	if secretInstallerSetSigningKey != tc.Spec.GenerateSigningSecret {
-		manifest := r.manifest
-		manifest = manifest.Filter(mf.ByKind("Secret"))
+	// Check for x509 keys
```

Comment:
Can you please add more details in the comment like why we are checking for x509 keys